<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>child</class>
 <widget class="QMainWindow" name="child">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>历史记录显示</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>251</width>
      <height>771</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>21</y>
       <width>251</width>
       <height>751</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>801</width>
      <height>771</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>791</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>590</y>
       <width>791</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
